README.md

```diff
@@ -30,7 +30,22 @@ Pravega Presto connector has a set of unit tests that can take several minutes t
 
     ./gradlew test
 
-## Running Presto
+## Installing Presto
```

Comment:
There are possibly multiple ways to get the server started, I'll describe in my comments the one I followed based on the presto deploy documentation. I find it more natural to use that as a reference as it is the first I encountered when searching. We can of course describe other ways, but I'd suggest that we use the way documented in:

https://prestodb.io/docs/current/installation/deployment.html

as a reference.

Reply:
I don't understand.  The docs say to use the launcher command to start the server.  That's what I said.